File: SagaiSambaandh/SagaiSambaandh/ios/SagaiSambaandh/Views/ChatView.swift
import SwiftUI

struct ChatView: View {
    @State private var selectedSubTab: Int = 0
    
    var body: some View {
        VStack(spacing: 0) {
            // Top Toolbar
            HStack {
                Spacer()
                Text("Chat")
                    .font(BrandFonts.displayBold(size: 20))
                    .foregroundColor(.lightGold)
                Spacer()
            }
            .padding()
            .background(Color.deepMaroon)
            
            // Sub Tabs Selection
            Picker("SubTabs", selection: $selectedSubTab) {
                Text("All Chats").tag(0)
                Text("Unread").tag(1)
            }
            .pickerStyle(SegmentedPickerStyle())
            .padding(.horizontal)
            .padding(.vertical, 8)
            .background(Color.deepMaroon)
            
            Spacer()
            
            // Empty State
            VStack(spacing: 20) {
                ZStack {
                    Circle()
                        .fill(Color.deepMaroon)
                        .frame(width: 100, height: 100)
                    
                    Image(systemName: "bubble.left.and.bubble.right.fill")
                        .font(.system(size: 40))
                        .foregroundColor(.lightGold)
                }
                
                Text("No Recent Chats")
                    .font(BrandFonts.displayBold(size: 18))
                    .foregroundColor(.lightGold)
                
                Text("Initiate contacts with compatible profiles or verify your profile to unlock custom matchmaking recommendations.")
                    .font(BrandFonts.body(size: 13))
                    .foregroundColor(.sandstoneIvory.opacity(0.7))
                    .multilineTextAlignment(.center)
                    .padding(.horizontal, 32)
            }
            
            Spacer()
        }
        .background(Color.deepMaroon.edgesIgnoringSafeArea(.all))
    }
}

File: android/SagaiSambaandh/app/src/main/java/com/sagaisambaandh/ui/screens/ChatView.kt
package com.sagaisambaandh.ui.screens

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.filled.Chat
import androidx.compose.material.icons.filled.ChatBubbleOutline
import androidx.compose.material.icons.filled.Info
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.text.font.FontFamily
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp
import com.sagaisambaandh.ui.theme.*

@OptIn(ExperimentalMaterial3Api::class)
@Composable
fun ChatView(modifier: Modifier = Modifier) {
    var selectedSubTab by remember { mutableStateOf(0) } // 0 = All, 1 = Unread

    Column(
        modifier = modifier
            .fillMaxSize()
            .background(MaterialTheme.colorScheme.background)
    ) {
        // Top Toolbar
        CenterAlignedTopAppBar(
            title = {
                Text(
                    text = "Chat",
                    fontFamily = FontFamily.Serif,
                    fontWeight = FontWeight.Bold,
                    fontSize = 20.sp,
                    color = LightGold
                )
            },
            colors = TopAppBarDefaults.centerAlignedTopAppBarColors(
                containerColor = DeepMaroon
            )
        )

        // Sub Tabs
        TabRow(
            selectedTabIndex = selectedSubTab,
            containerColor = DeepMaroon,
            contentColor = LightGold,
            divider = {},
            modifier = Modifier.fillMaxWidth()
        ) {
            Tab(
                selected = selectedSubTab == 0,
                onClick = { selectedSubTab = 0 }
            ) {
                Text(
                    text = "All Chats",
                    fontSize = 13.sp,
                    fontWeight = FontWeight.Bold,
                    color = if (selectedSubTab == 0) LightGold else SandstoneIvory.copy(alpha = 0.6f),
                    modifier = Modifier.padding(vertical = 12.dp)
                )
            }
            Tab(
                selected = selectedSubTab == 1,
                onClick = { selectedSubTab = 1 }
            ) {
                Text(
                    text = "Unread",
                    fontSize = 13.sp,
                    fontWeight = FontWeight.Bold,
                    color = if (selectedSubTab == 1) LightGold else SandstoneIvory.copy(alpha = 0.6f),
                    modifier = Modifier.padding(vertical = 12.dp)
                )
            }
        }

        Spacer(modifier = Modifier.height(60.dp))

        // Empty State Content
        Column(
            modifier = Modifier
                .fillMaxWidth()
                .padding(24.dp),
            horizontalAlignment = Alignment.CenterHorizontally,
            verticalArrangement = Arrangement.Center
        ) {
            Box(
                modifier = Modifier
                    .size(100.dp)
                    .background(DeepMaroon, shape = RoundedCornerShape(50.dp)),
                contentAlignment = Alignment.Center
            ) {
                Icon(
                    imageVector = Icons.Default.ChatBubbleOutline,
                    contentDescription = null,
                    tint = LightGold,
                    modifier = Modifier.size(48.dp)
                )
            }

            Spacer(modifier = Modifier.height(24.dp))

            Text(
                text = "No Recent Chats",
                color = LightGold,
                fontSize = 18.sp,
                fontWeight = FontWeight.Bold,
                fontFamily = FontFamily.Serif,
                textAlign = TextAlign.Center
            )

            Spacer(modifier = Modifier.height(10.dp))

            Text(
                text = "Initiate contacts with compatible profiles or verify your profile to unlock custom matchmaking recommendations.",
                color = SandstoneIvory.copy(alpha = 0.7f),
                fontSize = 13.sp,
                lineHeight = 18.sp,
                textAlign = TextAlign.Center,
                modifier = Modifier.padding(horizontal = 16.dp)
            )

            Spacer(modifier = Modifier.height(30.dp))
        }
    }
}
